sphinx/source/docs/releases/2.3.0.rst

```diff
@@ -13,6 +13,33 @@ And several other bug fixes and docs additions. For full details see the
 `Migration Guide <releases.html#release-2-3-0-migration>`__
 -----------------------------------------------------------
 
+Properties are non-nullable by default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Previously all properties accepted ``None`` (``null`` in bokehjs) as a value.
+In this release this was changed and ``None`` is accepted only by properties
+marked as nullable, e.g. ``Nullable(Int)`` accepts integers and ``None``. All
+exsisting models and their properties were updated, so there will be no
+difference from users' perspetive. However, extensions' creators are affected,
+who will have to update their models.
```

Comment:
We can edit this in another docs PR before the release but I think it would be good to highlight:

* that setting to None could often be a cause of silent errors, this just makes thos problems louder and earlier
* extension updates are only required if an extension property can and should be set to None, correct? If a property id not intended to be Nullablle then there should be no change needed? If so let;s bo explicit about things.